services/document-manager/backend/Dockerfile

```diff
@@ -17,4 +17,6 @@ EXPOSE 5001
 
 RUN mkdir -p /tmp/celery
 
+RUN mkdir -p /tmp/spatial
```

Comment:
I see that this doesn't match exactly the folder the assets are downloaded to. We should probably add the missing `/` to the docman paths

![image](https://github.com/user-attachments/assets/1dca2290-a081-4aa3-a5bf-9bc18d72f30e)

Reply:
I was working with this in my PR too, but from what I can tell the folder doesn't get persisted to the resulting container after this command (/tmp/celery also doesn't exist if you exec into the container after creation)

Reply:
In my PR I added the creation in the init file for docman.

Reply:
I just took this stuff out from both Dockerfiles. 👍